CPT331.Core/ObjectModel/Coordinate.cs

```diff
@@ -91,27 +89,10 @@ public static bool IsValid(double latitude, double longitude)
 			return (IsLatitudeValid(latitude) && IsLongitudeValid(longitude));
 		}
 
-		public override string ToString()
+		public static Coordinate FromValues(double latitude, double longitude)
 		{
-			return $"Latitude = {_latitude}, Longitude = {_longitude}";
-		}
-
-        public static bool TryCoordinate(double latitude, double longitude, out Coordinate output)
-        {
-			if (IsValid(latitude, longitude) == true)
-			{
-				output = new Coordinate(latitude, longitude);
-			}
-			else
-			{
-				output = null;
-			}
-
-			return (output != null);
-        }
+			Coordinate coordinate = null;
 
-        private void ValidateCordinate(double latitude, double longitude)
-        {
 			if (IsValid(latitude, longitude) == false)
```

Comment:
Consolidated all the validation into this factory method - this is now the only way you can new up a type instance.